### DISCUSSION TITLE
how to delete the child through an update on the parent?

### DISCUSSION THREAD

**Discussion by rntjr** [prisma/prisma#15582] [Q&A]
I have an object like this:

```json
{
	"name":"Renato Franco Boachaques Junior",
	"birthDate": "2000-04-10",
	"authorities": [
		{"name":"ADMIN"},
		{"name":"READ_REPORT"}
	]
}
```

But i need update it and delete one children, like this:

```json
{
	"name":"Renato Franco Boachaques Junior",
	"birthDate": "2000-10-15",
	"authorities": [
		{"name":"READ_REPORT"}
	]
}
```

what is the best approach to this?

**Comment by nurul3101** [comment]
Hey @rntjr 👋 
Which database are you using?
Have you tried using the [set](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#set-1) operator to override your array?

**Reply by rntjr** [reply]
im use postgres, i just a wanted an example

**Reply by rntjr** [reply]
I needed an example that works with both an element and objects

**Reply by pantharshit00** [reply]
Can you please share your schema? That way we can provide you with an solution because right now it is not clear whether `authorities` is a JSON field or a relationship.

**Reply by vladi-stevanovic** [reply]
👋 Hi @rntjr , did you perhaps have a chance to check Harshit's question? Let us know if you can share your schema so we can continue to investigate here!

Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
